src/SignalR/common/Http.Connections/src/Internal/AuthorizeHelper.cs

```diff
@@ -1,69 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System.Collections.Generic;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Authentication;
-using Microsoft.AspNetCore.Authorization;
-using Microsoft.AspNetCore.Authorization.Policy;
-using Microsoft.Extensions.DependencyInjection;
-
-namespace Microsoft.AspNetCore.Http.Connections.Internal
-{
-    internal static class AuthorizeHelper
```

Comment:
It makes me super happy to see this go away. We basically have 3 version of this code rn, and this cuts it down to two.